From: bitcoffee<854182924@qq.com>
Reason: repair libgrpc++_test_config.so lost dependence on glfags

diff -urN grpc/cmake/gflags.cmake grpc_new/cmake/gflags.cmake
--- grpc/cmake/gflags.cmake	2020-08-25 10:00:46.356802187 +0800
+++ grpc_new/cmake/gflags.cmake	2020-08-25 20:00:19.296009625 +0800
@@ -25,8 +25,8 @@
 elseif(gRPC_GFLAGS_PROVIDER STREQUAL "package")
   # Use "CONFIG" as there is no built-in cmake module for gflags.
   find_package(gflags REQUIRED CONFIG)
-  if(TARGET gflags::gflags)
-    set(_gRPC_GFLAGS_LIBRARIES gflags::gflags)
+  if(TARGET gflags)
+    set(_gRPC_GFLAGS_LIBRARIES gflags)
   endif()
   set(_gRPC_FIND_GFLAGS "if(NOT gflags_FOUND)\n  find_package(gflags CONFIG)\nendif()")
 elseif(gRPC_GFLAGS_PROVIDER STREQUAL "none")
